molo/profiles/templates/profiles/editprofile.html

```diff
@@ -3,29 +3,27 @@
 {% get_settings %}
```

Comment:
Please apply the things that I've been asking you to do in `change_password` in this file as well.